client/go/outline/electron/main.go

```diff
@@ -98,7 +87,10 @@ func main() {
 
 	setLogLevel(*args.logLevel)
 
-	client, err := newShadowsocksClientFromArgs()
+	if len(*args.transportConfig) == 0 {
+		printErrorAndExit(errors.New("transport config missing"), 1)
```

Comment:
Suggest to use `platerrors.New` instead of `errors.New`, as it's the interface between Go and TypeScript.

Reply:
Done. There's no `platerrors.New` though.